src/components/RespondentsTable/RespondentsTable.utils.ts

```diff
@@ -1,23 +1,21 @@
 import { UserData, UsersItems } from 'redux/modules';
 
-export const prepareAppletUsersData = (id: string, items?: UsersItems['items']) =>
-  items &&
-  items.reduce((acc: UserData[] | null, currentValue) => {
-    if (currentValue[id] && acc) {
-      return acc.concat(currentValue[id]);
-    }
+export const prepareUsersData = (items?: UsersItems['items'], id?: string) =>
+  id && items
```

Comment:
is this different from  prepareManagersData?

Reply:
it has different types